test/lessify.js

```diff
@@ -53,7 +53,7 @@ test('should throw on invalid less', function(t) {
     , s = lessify('test.less');
 
   s.write('.}');
-  t.throws(function() { s.end(); }, new RegExp('missing opening `\\{`: test\\.less\\(1\\)'), 'should throw on invalid less');
+  t.throws(function() { s.end(); }, new RegExp('.*'), 'should throw on invalid less');
```

Comment:
Unless I'm missing something here, it can't be done with Tape.  The second argument to throws() assertion method has to be a RegExp, and that RegExp is applied to whatever was thrown.  But less.render() is throwing an object, rather than a string, so there's nothing we can say in a RegExp that will match it.